Fix a bug on Python 3.11 that overriding an existing command from plugins raises an error.
